plugins/git/README.md

```diff
@@ -10,178 +10,178 @@ plugins=(... git)
-| g                    | git                                                                                                                              |
-| ga                   | git add                                                                                                                          |
-| gaa                  | git add --all                                                                                                                    |
-| gapa                 | git add --patch                                                                                                                  |
-| gau                  | git add --update                                                                                                                 |
-| gav                  | git add --verbose                                                                                                                |
-| gap                  | git apply                                                                                                                        |
-| gapt                 | git apply --3way                                                                                                                 |
-| gb                   | git branch                                                                                                                       |
-| gba                  | git branch -a                                                                                                                    |
-| gbd                  | git branch -d                                                                                                                    |
+| Alias                | Command                                                                                                                                                           |
+| :------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| grt                  | cd "$(git rev-parse --show-toplevel \|\| echo .)"                                                                                                                 |
+| ggpnp                | ggl && ggp                                                                                                                                                        |
+| ggpur                | ggu                                                                                                                                                               |
+| g                    | git                                                                                                                                                               |
```

Comment:
you sure about the alphabeticall order? ^^ (grt before ggpnp before g 😮 )

Reply:
The alphabetical order is on the  **Command** column, not the **Alias**.

Reply:
@cmalard Now what!? Lets merge!? 😄

Reply:
The maintainers have a huge backlog of PRs; that's why there hasn't been much activity.

If you _really_ want it merged, try asking on the discord server.

Reply:
@TheTechRobo thanks for the hint!
I will definitely try the Discord server!
I would love to merge it. This is my first contribution to an open source project!

Reply:
Congrats on your first contribution!! :D 

Yeah, I do believe it'll be useful. Much easier to read!

Reply:
🙌